components/board/genericlinux/data.go

```diff
@@ -92,75 +75,44 @@ func readIntFile(filePath string) (int, error) {
 	return int(resultInt64), err
 }
 
-func getGpioChipDefs(pinDefs []PinDefinition) (map[string]gpioChipData, error) {
-	gpioChipsInfo := map[string]gpioChipData{}
-	sysfsPrefixes := []string{"/sys/devices/", "/sys/devices/platform/", "/sys/devices/platform/bus@100000/"}
+func getGpioChipDefs(pinDefs []PinDefinition) (map[int]gpioChipData, error) {
```

Comment:
From our discussion with James, I thought the plan was to force the names of pins to be strings even though all of them until now had been numeric. Why did the keys in the map change from strings to ints? Won't we just need to change that back again next PR?

edit: after reading further, I suspect it's a map from ngpio values to chip data. Put in a comment here at the top explaining what the return value is.